src/game/client/anim2d.cpp

```diff
@@ -30,7 +30,8 @@ static const char *s_Anim2DModeNames[] = {
 const FieldParse Anim2DTemplate::s_anim2dFieldParseTable[] = { { "NumberImages", &Parse_Num_Images, nullptr, 0 },
     { "Image", &Parse_Image, nullptr, 0 },
     { "Image_Sequence", &Parse_Image_Sequence, nullptr, 0 },
-    { "AnimationMode", &INI::Parse_Index_List, s_Anim2DModeNames, offsetof(Anim2DTemplate, m_animMode) },
+    // { "AnimationMode", &INI::Parse_Index_List, s_Anim2DModeNames, offsetof(Anim2DTemplate, m_animMode) },
+    FIELD_PARSE_INDEXLIST("AnimationMode", s_Anim2DModeNames, Anim2DTemplate, m_animMode),
```

Comment:
game lib will now assert on compilation if `m_animMode` was not an enum and `s_Anim2DModeNames` was not a C string array.